### PR TITLE
Update repo

### PR DESCRIPTION
**Pull request by survived**
Update repo to be aligned with other projects:
* Add badges
* Add contribution and security guidelines
* Add changelog check to CI
* Update repo link in Cargo.toml

### CODE REVIEW

File: .github/changelog.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+m_branch=m;
+changelog_file=CHANGELOG.md;
+
+# fetch master since we might be in a shallow clone
+git fetch origin "$m_branch:$m_branch" --depth=1
+
+changed=0;
+for log in */"$changelog_file"; do
```
Comment (maurges) [inline .github/changelog.sh]: Changelog is in the root of this repo, not a subdirectory, so the leading `*` shouldn't be here

**Review by maurges — CHANGES_REQUESTED** [review]

File: .github/changelog.sh (hunk 1 above)
Comment (survived) [reply .github/changelog.sh]: Hmm, I can remove the star, but I would prefer to have one script for all projects

Comment (survived) [reply .github/changelog.sh]: If I do `for log in "$changelog_file" */"$changelog_file"; do`, this should work, right?

Comment (maurges) [reply .github/changelog.sh]: I remember I had some problems with it, but I'm trying it now and it works. Let's do it like this

Comment (survived) [reply .github/changelog.sh]: Ehh, I will need to update all repos... This is fun

Comment (maurges) [reply .github/changelog.sh]: You don't have to, we can leave them as is as it already works

Comment (survived) [reply .github/changelog.sh]: Yes, but then the next time I need the latest script, I'll have hard time remembering which repo it was with the latest version

Comment (survived) [reply .github/changelog.sh]: And actually, one of the repos with changelog in the root was using the script that works with changelogs in `*/`. So it's a good thing to keep everything in sync.

**Review by maurges — APPROVED** [review]

**Pull request merged**